chain/x/oracle/keeper/reporter.go

```diff
@@ -33,3 +33,22 @@ func (k Keeper) RemoveReporter(ctx sdk.Context, val sdk.ValAddress, addr sdk.Acc
 	ctx.KVStore(k.storeKey).Delete(types.ReporterStoreKey(val, addr))
 	return nil
 }
+
+// GetReporters returns the reporter list of the given validator.
+func (k Keeper) GetReporters(ctx sdk.Context, val sdk.ValAddress) (reporters []sdk.AccAddress) {
+	// Appends self reporter of validator to the list
+	selfReporter := sdk.AccAddress(val)
+	reporters = append(reporters, selfReporter)
+
+	store := ctx.KVStore(k.storeKey)
+	storeKeyPrefix := append(types.ReporterStoreKeyPrefix, val.Bytes()...)
```

Comment:
Move this line to `types/key.go` create a new function `ValidatorReporterPrefixKey` to generate this key